spacewalk/config/usr/share/man/man5/rhn.conf.5

```diff
@@ -235,6 +235,13 @@ When importing packages, skip non-essential data that can make the import faster
 .B Default:
 0
 
+.TP
+.B "java.smtp_server"
+This parameter defines the server Satellite will use as its SMTP server.
+.IP
+.B Default:
+none
```

Comment:
@csheremeta I think you should add the default as a localhost as we can see here on the code. 

```bash
↪ grep -R WEB_SMTP * 
common/conf/ConfigDefaults.java:    public static final String WEB_SMTP_SERVER = "java.smtp_server";
common/messaging/SmtpMail.java:        smtpHost = c.getString(ConfigDefaults.WEB_SMTP_SERVER, "localhost");
```